installation/geckodriver.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+version=${1:-latest}
+
+install_dir="/usr/local/bin"
+base_url="https://github.com/mozilla/geckodriver/releases/download"
+
+if [[ ${version} == "latest" ]]; then
+    json=$(curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest)
```

Comment:
I've played with this before and hit issues with rate limits, but if this is working then great!

Reply:
I hit that too, took me hours to realize, hence the fallback 😂